src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/LSPRazorProjectHost.cs

```diff
@@ -104,25 +105,34 @@ public Task UnloadAsync()
                     clientName = clientNameValue.ToString();
                 }
 
+                var disableUserExperience = false;
+                if (client.Metadata.TryGetValue("DisableUserExperience", out var disableUserExperienceValue))
```

Comment:
Should this be `nameof(IIsUserExperienceDisabledMetadata.IsUserExperienceDisabled)`? Also, did anything in particular drive this inclusion here or were you just cleaning things up as you saw it?

Reply:
No, the name of the attribute in the metadata is actually Disable user experience, so nameof(...IsUserExperienceDisabled) doesn't find anything.

Yes this is required, otherwise this metadata is dropped when activating the ilanguageclient.  This causes for example the liveshare c# lsp server to erroneously provide results for local c# files.